client/my-sites/plugins/use-preinstalled-premium-plugin/index.js

```diff
@@ -28,18 +28,24 @@ export default function usePreinstalledPremiumPlugin( pluginSlug ) {
 		if ( ! preinstalledPremiumPlugin || ! selectedSiteId ) {
 			return false;
 		}
+
 		// Always active on simple sites
```

Comment:
I think this should check the `jetpack_search_enabled` setting for Simple sites to avoid the discrepancy I mentioned earlier. This is how is done on the Settings > Performance screen:

https://github.com/Automattic/wp-calypso/blob/ce8d3f799c9ee0baf850f3ef86fc57d1f73d09f6/client/my-sites/site-settings/search.jsx#L261